nodogsplash2/files/nodogsplash.config

```diff
@@ -14,27 +14,34 @@ config nodogsplash
 
   # The network the users are connected to
   option network 'lan'
-  option gatewayname 'LEDE Nodogsplash'
+  option gatewayname 'OpenWrtLEDE Nodogsplash'
   option maxclients '250'
-  option clientidletimeout '1200'
+  #Client timeouts in minutes
+  option clientidletimeout '120'
+  option clientforcetimeout '1440'
+
 
   # Your router may have several interfaces, and you
   # probably want to keep them private from the network/gatewayinterface.
   # If so, you should block the entire subnets on those interfaces, e.g.:
-  list authenticated_users 'block to 192.168.0.0/16'
-  list authenticated_users 'block to 10.0.0.0/8'
+  # list authenticated_users 'block to 192.168.0.0/16'
+  # list authenticated_users 'block to 10.0.0.0/8'
```

Comment:
Could it be that we lock out the user?
User flashes his device with OpenWrt and NDS and tries to ssh to the device,
but this might not work this NDS redirects all traffic to the flash site.
This must be tested...

Reply:
I have tested and is fine. The two lines (or similar) are designed to block access to other subnets that might also reside on the router eg an office network, and provide a firewall function between public and private LAN subnets.

Reply:
The user to router section is set to allow ssh, dhcp, dns and http:

# Allow ports for SSH/Telnet/DNS/DHCP/HTTP/HTTPS
  list users_to_router 'allow tcp port 22'
  list users_to_router 'allow tcp port 23'
  list users_to_router 'allow tcp port 53'
  list users_to_router 'allow udp port 53'
  list users_to_router 'allow udp port 67'
  list users_to_router 'allow tcp port 80'

Reply:
ok :)